src/store/modules/order/actions.ts

```diff
@@ -46,10 +46,10 @@ const actions: ActionTree<OrderState, RootState> = {
     if (item) {
       item.quantityAccepted = item.quantityAccepted ? parseInt(item.quantityAccepted) + 1 : 1;
       commit(types.ORDER_CURRENT_UPDATED, state.current )
-      showToast(translate("Scanned successfully.", { itemName: payload }))
-    } else {
-      showToast(translate("Scanned item is not present within the shipment:", { itemName: payload }))
+      return { isUpdated: true, itemName: payload }
```

Comment:
No need to return `itemName` from here, as we already have `itemName` in the calling method.